testing/presto-testng-services/src/main/resources/META-INF/services/org.testng.ITestNGListener

```diff
@@ -0,0 +1 @@
+com.facebook.presto.testng.services.ReportMultiThreadedBeforeOrAfterMethod
```

Comment:
We don't use this file here? https://github.com/prestodb/presto/blob/master/presto-main/src/test/resources/META-INF/services/org.testng.ITestNGListener

I don't think this actually works as intended, but not 100% sure. I suspect only the unit test actually incorporates this listener. The rest won't be using it.

Reply:
There will be cyclic dependencies if put the listeners under presto-main so the listeners must reside in a separate project.
I debugged into the listeners and id really works. I will consolidate all the listeners into one place.